ui/app/src/components/workspaces/WorkspaceOperationsPanel.tsx

```diff
@@ -0,0 +1,100 @@
+import { DefaultPalette, IStackItemStyles, IStackStyles, Stack } from "@fluentui/react";
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Operation } from '../../models/operation';
+import { Workspace } from '../../models/workspace';
+import { ApiEndpoint } from '../../models/apiEndpoints';
+import { HttpMethod, ResultType, useAuthApiCall } from '../../useAuthApiCall';
+import config from '../../config.json';
+
+
+
+interface WorkspaceOperationPanelProps {
+    workspace: Workspace
+}
+
+interface WorkspaceOperationPanelItemProps {
+    header: String,
+    val: String
+}
+
+const WorkspaceOperationPanelItem: React.FunctionComponent<WorkspaceOperationPanelItemProps> = (props: WorkspaceOperationPanelItemProps) => {
+
+
+
+    const stackItemStyles: IStackItemStyles = {
+        root: {
+            padding: 5,
+            width: 150,
+            color: DefaultPalette.neutralSecondary
+        }
+    }
+    return(
+        <>
+            <Stack wrap horizontal>
+                <Stack.Item grow styles={stackItemStyles}>
+                    {props.header}
+                </Stack.Item>
+                <Stack.Item grow styles={stackItemStyles}>
+                    : {props.val}
+                </Stack.Item>
+            </Stack>
+        </>
+    );
+}
+
+export const WorkspaceOperationsPanel: React.FunctionComponent<WorkspaceOperationPanelProps> = (props: WorkspaceOperationPanelProps) => {
+    const apiCall = useAuthApiCall();
+    const { workspaceId } = useParams();
+    const [workspaceOperations, setWorkspaceOperations] = useState([] as Array<Operation>)
+    
+
+    const getOperations = async () => {
+        let ws = props.workspace && props.workspace.id ?
+        props.workspace :
+        (await apiCall(`${ApiEndpoint.Workspaces}/${workspaceId}`, HttpMethod.Get)).workspace;
+
+        await apiCall(`${ApiEndpoint.Workspaces}/${workspaceId}`, HttpMethod.Get, ws.properties.app_id, undefined, ResultType.None, (roles: Array<string>) => {
```

Comment:
don't need this, we have set the roles already in the parent

Reply:
👍